src/components/BodyComponent.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState , useEffect} from 'react'
+import CardComponent from './CardComponent'
+import SearchBar from './SearchBar'
+import {Link} from 'react-router-dom'
+const BodyComponent = () => {
+   const [searchData, setsearchData] = useState([])
+   useEffect(()=>{
+    fetchTeamData()
+   },[])
+   async function fetchTeamData(){
+      const teamData = ['nam334','BhaveshSSingh','gk9848970']
+      const arr = []
+      teamData.map(async (teamMemberId,i)=>{
+        const data =  await fetch(`https://api.github.com/users/${teamMemberId}`)
```

Comment:
can take this api call to a file under api folder

Reply:
noted